js/main.js

```diff
@@ -1,6 +1,44 @@
 var $cardList = document.querySelector('.card-list');
+var $genres = document.querySelector('.genres');
 
-renderPage1Movies();
+renderPopularMovies1();
+
+$genres.addEventListener('click', function (e) {
+  var highlightBtn = document.querySelector('.highlight');
+  highlightBtn.className = '';
+  e.target.className = 'highlight';
+  removeAllChildren($cardList);
+  if (e.target.outerText === 'Popular') {
+    renderPopularMovies1();
+  }
+  if (e.target.outerText === 'Action') {
+    renderActionMovies1();
+  }
+  if (e.target.outerText === 'Thriller') {
+    renderThrillerMovies1();
+  }
+  if (e.target.outerText === 'Drama') {
+    renderDramaMovies1();
+  }
+  if (e.target.outerText === 'Adventure') {
+    renderAdventureMovies1();
+  }
+  if (e.target.outerText === 'Comedy') {
+    renderComedyMovies1();
+  }
+  if (e.target.outerText === 'Horror') {
+    renderHorrorMovies1();
+  }
+  if (e.target.outerText === 'Animation') {
+    renderAnimationMovies1();
+  }
+  if (e.target.outerText === 'Romance') {
+    renderRomanceMovies1();
+  }
+  if (e.target.outerText === 'Mystery') {
+    renderMysteryMovies1();
+  }
```

Comment:
Instead of needing a conditional for each option, you should add a `data-` attribute to each of the buttons for each genre, with the number that correlates to that genre from the API as its value, then just pass that number as an argument to a function that can construct the URL to get movies for that specific genre. One function that can build the URL for any genre.